glide.yaml

```diff
@@ -50,9 +50,9 @@ import:
   version: a8983c062be4b565d723c478922d7736e04fdba4
 - package: github.com/Masterminds/semver
   version: v1.3.1
+- package: github.com/imdario/mergo
+  version: ^0.2.2
 testImport:
 - package: github.com/onsi/gomega
 - package: github.com/onsi/ginkgo
-  version: v1.3.1
 - package: github.com/kelseyhightower/envconfig
-  version: ~1.3.0
```

Comment:
@JiangtianLi just add back those version strings so the diff is clean for these two packages, but don't worry about re-running glide or anything